src/olympia/accounts/utils.py

```diff
@@ -71,8 +73,12 @@ def fxa_login_url(config, state, next_path=None, action=None,
         if id_token:
             query['prompt'] = 'none'
             query['id_token_hint'] = id_token
-    return '{host}/authorization?{query}'.format(
-        host=settings.FXA_OAUTH_HOST, query=urlencode(query))
+    if use_fake_fxa():
+        base_url = reverse('fake-fxa-authorization')
+    else:
+        base_url = '{host}/authorization'.format(host=settings.FXA_OAUTH_HOST)
+    return '{base_url}?{query}'.format(
+        base_url=base_url, query=urlencode(query))
```

Comment:
Not super pretty but the reason I'm doing this instead of overriding `FXA_CONFIG` for local envs is that I wanted to make it work regardless of the `config` used, and also make it easy to switch back to the real config if needed.